src/commands/ReminderCommand/ReminderCommand.js

```diff
@@ -0,0 +1,83 @@
+const BaseCommand = require('../../utils/structures/BaseCommand');
+const momentTimezone = require('moment-timezone');
+const { MessageCollector } = require('discord.js');
+
+const scheduledSchema = require('../../models/scheduled-schema');
+
+module.exports = class ReminderCommand extends BaseCommand {
+  constructor() {
+    super('schedule', 'textAnswers', []);   // change the name to remind later
+  }
+
+  async run(client, message, args) {    // expectedArgs: '<Channel tag> <YYYY/MM/DD> <HH:MM> <"AM" or "PM"> <Timezone>',
+    const { mentions, guild, channel } = message; 
+    
+    const userID = message.author.id;   // userID will be needed to tag the person on the reminder
+
+    const targetChannel = mentions.channels.first();
+    
+    if(!targetChannel){
+        message.reply('Please tag a channel to send your message in');
+        return;
+    }
+
+    // Remove the 1st argument from the args array
+    args.shift();
+
+    let [date, time, clockType, timeZone] = args;
+    
+    if(clockType !== 'AM' && clockType !== 'PM'){
```

Comment:
convert clockType to uppercase to allow the user to specify "am" or "pm" as well